shard-operator/src/main/resources/application.properties

```diff
@@ -15,9 +15,9 @@ event-bridge.default.kafka.client.id=${KAFKA_CLIENT_ID:default-id}
 event-bridge.default.kafka.client.secret=${KAFKA_CLIENT_SECRET:default-secret}
 event-bridge.default.kafka.security.protocol=${KAFKA_SECURITY_PROTOCOL:SASL_SSL}
 
-event-bridge.authorization.sso.client-id==${EVENT_BRIDGE_SSO_CLIENT_ID:event-bridge}
-event-bridge.authorization.sso.url=${EVENT_BRIDGE_SSO_URL:http://localhost:8180/auth/realms/event-bridge-fm/protocol/openid-connect/token}
-event-bridge.authorization.sso.secret=${EVENT_BRIDGE_SSO_SECRET:secret}
+quarkus.oidc-client.auth-server-url=${EVENT_BRIDGE_SSO_URL:https://localhost:8180/auth/realms/event-bridge-fm}
```

Comment:
`https` -> `http` ;)

Reply:
hmmm, are we not using https?

Reply:
`https` on localhost?

Reply:
Or better... our keycloak instance is not exposed with `https` by docker compose. So it wont connect